docs/api-reference.md

```diff
@@ -0,0 +1,481 @@
+## API
+
+
+# SILACAK API DOCS
```

Comment:
```suggestion
# API

## Silacak API Docs
```

- Nitpick: Please use title case instead of all caps for Markdown headings.
- Extra line removed.

Please also re-check all headings and make sure they are at the correct level (h1-h4), for accessibility reasons.